### PR TITLE
Change contact for re-review requests in "affiliated decision template"

### PR DESCRIPTION
**Pull request by hamogu**
Point of contact for this should now be the editors and I believe this case is so rare (not sure it has happened yet) that "drop us an email" is good enough. No issue or PR needed.

### CODE REVIEW

**Comment by astrofrog** [comment]
👍

File: affiliated/affiliated_package_review_process.md [hunk 1]
```diff
@@ -339,7 +339,9 @@ respond to the comments and/or address any of them.
 If you have any follow-up questions or disagree with any of the comments above,
 leave a comment and we can discuss it here. At any point in future you can
 request a re-review of the package if you believe any of the scores should be
-updated - contact the coordination committee, and we’ll do a new review.
+updated - contact the affiliated package editors
+[listed on the Astropy team page](https://www.astropy.org/team),
+and we’ll do a new review.
```
Comment (pllim) [inline affiliated/affiliated_package_review_process.md:344]: Nitpick: More consistent with copy-edit rules set by our paid copy-editor this way, though this strictly is not related to your changes.
```suggestion
and we will do a new review.
```
